src/discover_cone.cpp

```diff
@@ -55,100 +50,86 @@ DiscoverCone::DiscoverCone() :
 	ROS_INFO("[DiscoverCone] PARAM odometry_topic_name: %s", odometry_topic_name_.c_str());
 }
 
+// Convert an Euler angle into a range of [0 .. 360).
 double DiscoverCone::normalizeEuler(double yaw) {
```

Comment:
It's better to use
http://wiki.ros.org/angles
http://docs.ros.org/api/angles/html/namespaceangles.html#a100ca493044236d7e099e08e78aab5bf

Rather than write your own. Why is this in degrees?